e2e/src/bin/historic_prices.rs

```diff
@@ -133,26 +135,32 @@ where
             estimate,
         }: Row,
     ) -> Result<()> {
-        let error = (estimate as f64 - price as f64).abs() / price as f64;
+        let price_estimate = estimate.unwrap_or_default();
```

Comment:
Do you still want to treat a missing estimate as 0.0?

Reply:
For the CSV file yes, but we don't count missing estimates towards the total error.